.github/workflows/pull-db-tests.yml

```diff
@@ -251,3 +263,17 @@ jobs:
         env:
           TAGS: bindata
           USE_REPO_TEST_DIR: 1
+
+  check-backend-coverage:
+    if: needs.files-changed.outputs.backend == 'true'
+    needs: [files-changed, test-pgsql, test-sqlite, test-unit, test-mysql5, test-mysql8, test-mssql]
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-go@v4
+      with:
+        go-version: ">=1.20"
+        check-latest: true
+    - uses: actions/download-artifact@v3
+    - run: go run build/gocoverage.go merge unit-test-coverage/coverage.out unit-test-coverage-gogit/coverage.out unit-test-coverage-integration/coverage.out > coverage.all
```

Comment:
Calls like this should be in the makefile so that coverage can be produced locally as well. Please don't lock us in into actions with commands like this.